src/Libraries/PythonNodeModels/PythonNodeModels.csproj

```diff
@@ -39,13 +39,13 @@
       <HintPath>..\..\packages\Newtonsoft.Json.8.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
       <Private>False</Private>
     </Reference>
-    <Reference Include="Python.Included, Version=3.7.3.4, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Python.Included.3.7.3.4\lib\netstandard2.0\Python.Included.dll</HintPath>
-      <Private>False</Private>
```

Comment:
please retain the copyLocal / private elements set to False where they were previously or we'll get multiple copies of these dlls potentially pulled into various output folders.